src/axolotl/loaders/adapter.py

```
@@ -165,19 +166,47 @@ def load_adapter(
+            return model, None
+
+        config = builder.build_config(model)
+
+        if config_only:
+            return None, config
+
+        if hasattr(model, "enable_input_require_grads"):
+            model.enable_input_require_grads()
+
+        model = builder.build_model(model, config)
+        return model, config
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)